capture/capture_aes.py

```diff
@@ -98,6 +98,7 @@ def setup(cfg: dict, project: Path):
         protocol = cfg["target"]["protocol"],
         pll_frequency = cfg["target"]["pll_frequency"],
         bitstream = cfg["target"].get("fpga_bitstream"),
+        force_program_bitstream = cfg["target"]["force_program_bitstream"],
```

Comment:
```suggestion
        force_program_bitstream = cfg["target"].get("force_program_bitstream"]),
```
I think we should use here the `.get()` functionality. The reason is that in the config files for the chip, we do not have the `force_program_bitstream` entry. Hence, when using `cfg["target"]["force_program_bitstream"]` the script would crash.

When using  `cfg["target"].get("force_program_bitstream"])` and the `force_program_bitstream` entry is not available (i.e., for the chip config), None is returned.

Reply:
Good point, let me do that! And thanks for the explanation. I wasn't sure what the exact purpose of the `.get` was.